src/renderer/components/hotbar/hotbar-add-command.tsx

```diff
@@ -25,10 +25,10 @@ import { HotbarStore } from "../../../common/hotbar-store";
 import { CommandOverlay } from "../command-palette";
 import { Input, InputValidator } from "../input";
 
-const uniqueHotbarName: InputValidator = {
+export const uniqueHotbarName: InputValidator = {
   condition: ({ required }) => required,
   message: () => "Hotbar with this name already exists",
-  validate: value => !HotbarStore.getInstance().getByName(value),
+  validate: value => !HotbarStore.getInstance().getByName(value.trim()),
```

Comment:
This doesn't need trim anymore since the value passed already has been trimmed (if trim is true on input).

Reply:
my bad, thank you